vip-helpers/vip-utils.php

```diff
@@ -120,11 +121,12 @@ function vip_powered_wpcom( $display = 'text', $before_text = 'Powered by ' ) {
  * @return string
  */
 function vip_powered_wpcom_url() {
+	$utm_term = isset( $_SERVER['HTTP_HOST'] ) ? $_SERVER['HTTP_HOST'] : '';
```

Comment:
Could use [Null Coalescing](https://www.php.net/manual/en/migration70.new-features.php) here:

```suggestion
	$utm_term = $_SERVER['HTTP_HOST'] ?? '';
```

PHPCS shouldn't complain about that.